recipes/fpgen/all/test_package/conanfile.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def configure(self):
+        v = int(str(self.settings.compiler.version).split(",")[0])
+        if self.settings.compiler == "gcc" and v < 10:
+            raise ConanInvalidConfiguration("Requires GCC > 10")
+        elif not self.settings.compiler == "gcc":
+            raise ConanInvalidConfiguration("Currently only GCC supported")
```

Comment:
probably, it should go to the recipe itself, not to the test_package's conanfile? should also make CI happy.

Reply:
@jay-tux You can use it as example: https://github.com/conan-io/conan-center-index/blob/be8842d5204c479acdbdfbdd7b12e56678e8a207/recipes/inversify-cpp/all/conanfile.py#L27

Reply:
Thank you so much! I tried to implement it.